master_key_rotation_for_disconnected_minions.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: Master Key Rotation for disconnected Minions
+- Start Date: 2021-05-27
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+There are several reasons to rotate keys:
+- Exposure or a compromised key
+- Security departments require key rotation on a periodic basis
+- It is common praxis to expire or revoke keys or certificates (e.g. [eset protect 1](https://help.eset.com/protect_admin/80/en-US/certificate_replacement.html), [eset protect 2](https://support.eset.com/en/kb6824-peer-certificate-is-going-to-expire-error-in-eset-security-management-center-7x))
```

Comment:
practice

Reply:
2 & 3 are really the same thing.

Reply:
Thank you, correcting both

Reply:
I meant the second and third bullet points are the same thing. It is common practice to rotate keys to limit future exposure.